docs/quickstart/www/config.php

```diff
@@ -0,0 +1,5 @@
+<?php
+return [
+    'consumerKey' => 'yis0TYCu7U9V4o7M',
+    'consumerSecret' => '74c5fd430cf1242a527f6223aebd42d30464be22',
```

Comment:
We need to add our comment here, maybe with more explanations as to what this configuration file is.

```
// XXX: The consumer secret should be in a properly secured credential store, 
// and *NEVER* checked into version control.```